src/endpoints/borrowed/delete/delete_by_id.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from fastapi import Depends, Path, status
+from sqlalchemy import and_, not_, select
+from sqlalchemy.orm import Session
+
+from src.dependencies import get_current_librarian, get_db
+from src.endpoints.borrowed.router_init import router
+from src.exceptions import custom_exception
+from src.models import all_models
+
+
+@router.delete(
+    "/{borrowed_id}", response_model=None, status_code=status.HTTP_204_NO_CONTENT
+)
+async def delete_borrowed(
```

Comment:
When a borrowed is deleted, the status of it's copy should be set to "available" if not already done, as the copy is no longer borrowed.

Reply:
Fixed Check Latest Commit.